### ISSUE TITLE
Instarest brings monsters who are behind a closed door

### ISSUE THREAD

**Issue by Sandman25DCSS** [jeremygurr/dcssca#667]
I entered vaults and now I kill one monster after another in a small room, they seem to get teleported in my view while still sleeping (I am uncannily stealthy)

**Comment by jeremygurr** [comment]
I have them brought in still sleeping so stealthy characters can still get the same advantage as if they had run away, rested, and come back and approached the sleeping monster. I could wake them but it would be a big nerf to stealth characters. 

Of course it shouldn't happen if the door is closed and there's no other way in. That's a bug.

**Comment by Sandman25DCSS** [comment]
Ok, hopefully it won't allow to split packs, currently it is risky to approach a large group of sleeping monsters in vanilla so moving them one by one in CA should not remove them from view of the group.

**Comment by jeremygurr** [comment]
Yeah the splitting of packs is definitely a problem. I will need to find a solution for that.
